src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to group CSS properties by type. Here, the box model properties (`width`, `height`, `border`, `border-radius`) should be grouped together, followed by the typography properties, and then the visual formatting properties (`display`, `flex-direction`, `align-items`, `padding`).